src/utils/encoreCatalogLogOutTimer.js

```diff
@@ -38,12 +38,14 @@ function EncoreCatalogLogOutTimer() {
       if (utils.hasCookie('VALID_DOMAIN_LAST_VISITED')) {
         utils.deleteCookie('VALID_DOMAIN_LAST_VISITED');
       }
-
-      // Completely log out the user
-      this.loadLogoutIframe(isTest);
```

Comment:
Yeah, I essentially agree that this should be dealt with by the other logout. 

I'm trying to think through the scenario where someone logs in, then closes their browser, and then revisits nypl.org 31 minutes later. Our timeout wouldn't delete anything then, well actually maybe it would — `logOutFromEncoreAndCatalogIn` would get set to run immediately I think... Either way, they would have a `VALID_DOMAIN_LAST_VISITED` cookie at that point, so the scenario this is correcting for would not be in play. 
Just thinking out loud I guess...

Reply:
> `logOutFromEncoreAndCatalogIn` would get set to run immediately I think

This is true